File: API/Angular/src/app/viewmodels/RoundViewModels/response-round-view-model.ts
import { ResponseUserRoundViewModel } from '../UserRoundViewModels/response-user-round-view-model';

export class ResponseRoundViewModel
{
  constructor(isOver, roundId, userRound)
  {
    this.roundId = roundId;
    this.isOver = isOver;
    this.userRound = userRound;
  }

  public isOver: boolean;
  public roundId: number;
  public userRound: Array<ResponseUserRoundViewModel>;
}

File: ViewModelsLayer/ViewModels/RoundViewModels/ResponseRoundViewModel.cs
using System.Collections.Generic;
using ViewModelsLayer.ViewModels.UserRoundViewModels;

namespace ViewModelsLayer.ViewModels.RoundViewModels
{
    public class ResponseRoundViewModel
    {
        public int RoundId { get; set; }
        public bool IsOver { get; set; }
        public List<ResponseUserRoundViewModel> UserRound { get; set; }
    }
}